src/app/modules/group/components/group-manager-list/group-manager-list.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component, Input, OnChanges, OnDestroy, SimpleChanges } from '@angular/core';
```

Comment:
The confirmation is also shown when editing management of another user than yourself:
See [this group](https://dev.algorea.org/branch/feature/manager-page-reducing-your-own-permissions/en/#/groups/by-id/609422240375608067;path=/details/managers) managers

Reply:
Fixed